Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM nvidia/cuda:9.0-cudnn7-devel
+
+SHELL ["/bin/bash", "-c"]
+
+RUN mkdir /input
+RUN mkdir /output
+RUN mkdir /license
+
+RUN apt-get update && apt-get install -y wget && apt-get install -y git
+RUN MINICONDA="Miniconda3-latest-Linux-x86_64.sh" && \
+    wget --quiet https://repo.continuum.io/miniconda/$MINICONDA && \
+    bash $MINICONDA -b -p /miniconda && \
+    rm -f $MINICONDA
+ENV PATH /miniconda/bin:$PATH
+
+# Add channels and build, then install
+RUN conda config --add channels omnia
+RUN conda update --yes -n base conda
+RUN conda config --add channels conda-forge
+RUN conda config --add channels omnia/label/dev
+RUN conda config --add channels openeye
+RUN pip install progressbar2
```

Comment:
Why can't you use the conda-installable `progressbar2` from conda-forge?
https://anaconda.org/conda-forge/progressbar2

Reply:
Changing now.